vite.config.js

```diff
@@ -0,0 +1,23 @@
+/// <reference types="vitest" />
+
+import { resolve } from "node:path";
+import { defineConfig } from "vite";
+
+export default defineConfig({
+	build: {
+		lib: {
+			// Could also be a dictionary or array of multiple entry points
+			entry: resolve(__dirname, "src/jdataview.js"),
+			name: "jDataView",
+			// the proper extensions will be added
+			fileName: "jdataview",
+			formats: ["es", "cjs", "umd"]
```

Comment:
If we're doing major bump anyway, these days I'd rather keep only ESM output. Just needs careful testing that our package.json works correctly in both browser and Node.

Reply:
Okay, I've got it set to just build es modules. I'm worried this might have some issues using in node, but I'll have to test that later.